doc/api/process.markdown

```diff
@@ -137,13 +137,29 @@ Here is an example that logs every unhandled rejection to the console
         // application specific logging, throwing an error, or other logic here
     });
 
-For example, here is a rejection that will trigger the `'unhandledRejection'`
+For example, here is a rejection that will trigger the 'unhandledRejection'
 event:
 
     somePromise.then(function(res) {
       return reportToUser(JSON.pasre(res)); // note the typo
     }); // no `.catch` or `.then`
 
+Here is an example of a coding pattern that will also trigger
+'unhandledRejection':
+
+    function SomeResource() {
+      // Initially set the loaded status to a rejected promise
+      this.loaded = Promise.reject(new Error('Resource not yet loaded!'));
```

Comment:
This is very unusual to me too. I tend to agree, especially since people who subscribe to `loaded` should be able to get a hook on when it happened.

Better yet - the construction should not pretend to perform IO.

I do think that we can find an example of an operational (not programmer) error happening with promises without it to document something that causes `unhandledRejection` and requires adding a `.catch` - for example reporting a non-critical event is pretty common, or sending an email in the background and not caring if it succeeded.